Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1491,6 +1491,19 @@ def lhegensim2018(fragment,howMuch):
 steps['LambdaBToLambdaMuMuToPPiMuMu_forSTEAM_13TeV_TuneCUETP8M1']=genvalid('LambdaBToLambdaMuMuToPPiMuMu_forSTEAM_13TeV_TuneCUETP8M1_cfi',step1GenDefaults)
 steps['BsToMuMu_forSTEAM_13TeV_TuneCUETP8M1']=genvalid('BsToMuMu_forSTEAM_13TeV_TuneCUETP8M1_cfi',step1GenDefaults)
 
+# Workflows for multiple concurrent lumi blocks
+
+step1LHEGenSimUp2018MultiLumi = merge ([{'--customise_commands': '"process.source.numberEventsInLuminosityBlock=cms.untracked.uint32(5)"',
+                                         '--nStreams': 4,
+                                         '--nConcurrentLumis': 2,
+                                         },step1LHEGenSimUp2018Default])
+
+def lhegensim2018ml(fragment,howMuch):
+    global step1LHEGenSimUp2018MultiLumi
```

Comment:
rather than declaring this variable global, I think it could just be defined inside this function

Reply:
@kpedro88 if we want to restrict its use to this specific function I agree, probably it makes sense to declare a global variable for the ```nConcurrentLumis``` and ```nStreams``` definition (which can be used also for data) and move the rest inside the function as you suggest